### PR TITLE
KIN-322 Add edit about us section view

### PR DESCRIPTION
**Pull request by Cristianpan**
[KIN-322](https://loopcrack.atlassian.net/browse/KIN-322) Medium [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=loopcrack-org&utm_campaign=badge_info)

### Problem
The admin user needs to edit the content of the About Us section of the public view through a form in the admin panel to have an easy way to edit its content.

### Solution

Make a page where the administrator user can edit the content of the about us section and can access it from the sidebar.

### Requirements checklist

- [x] It must be a form with the about us section contents.
- [x] It must allow to edit the following elements:
       - About us text 
       - About us video
       - About us image
- [x] It must have its own link on the sidebar
- [x] It must have a submit button to persist the changes.
- [x] It must display an alert when submitted the changes indicating success or failure on editing the about us section.

### Testing

- [x] Click on `Sobre Nosotros` on the sitebar and and you will be redirected to the page to edit the content of the about us section.
- [x] Submit the form and you can view the modal with a succesful message 

## Demo
Desktop view: 
image

Mobile view: 
image

[KIN-322]: https://loopcrack.atlassian.net/browse/KIN-322?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by LUGAMAFE** [comment]
hey @Cristianpan don't forget to send a demo image at least

**Review by CarlosJCD — APPROVED** [review]

**Review by CheloChidoone — APPROVED** [review]
Good Job @Cristianpan, It looks good, thanks for adding the demo images

**Review by LUGAMAFE — APPROVED** [review]

**Review by Mauriciocr207 — APPROVED** [review]

File: app/Views/admin/aboutUs/AboutUsEdit.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                    <div class="row">
+                        <div class="col-md-12">
+                            <div class="card">
+                                <div class="card-header">
+                                    <h5 class="card-title mb-0">Descripción de Sobre Nosotros</h5>
+                                </div>
+                                <div class="card-body">
+                                    <div class="hstack gap-3 align-items-start">
+                                        <div class="flex-grow-1">
+                                            <textarea class="form-control" rows="3" name="aboutUsText" required><?= old('aboutUsText') ?? ''?></textarea>
```
Comment (FernandoJoachin) [inline app/Views/admin/aboutUs/AboutUsEdit.php:68]: Are you planning to include the return of those values in a future PR, in other words, are you thinking of adding a 'withInput' afterwards in the controller?"
@Mauriciocr207

**Review by FernandoJoachin — APPROVED** [review]

File: app/Views/admin/aboutUs/AboutUsEdit.php (hunk 1 above)
Comment (Cristianpan) [reply app/Views/admin/aboutUs/AboutUsEdit.php:68]: That's right

**Pull request merged**